config/install.yaml

```diff
@@ -49,32 +49,30 @@ metadata:
 apiVersion: v1
 kind: ConfigMap
 metadata:
+  name: kyverno
+  namespace: kyverno
   labels:
-    app.kubernetes.io/component: kyverno
+    app.kubernetes.io/part-of: kyverno
     app: kyverno
     app.kubernetes.io/name: kyverno
-    app.kubernetes.io/part-of: kyverno
     app.kubernetes.io/version: latest
-  name: kyverno
-  namespace: kyverno
 data:
-  defaultRegistry: "docker.io"
   enableDefaultRegistryMutation: "true"
-  # resource types to be skipped by kyverno policy engine
+  defaultRegistry: "docker.io"
+  generateSuccessEvents: "false"
   resourceFilters: "[*,kyverno,*][Event,*,*][*,kube-system,*][*,kube-public,*][*,kube-node-lease,*][Node,*,*][APIService,*,*][TokenReview,*,*][SubjectAccessReview,*,*][SelfSubjectAccessReview,*,*][Binding,*,*][ReplicaSet,*,*][AdmissionReport,*,*][ClusterAdmissionReport,*,*][BackgroundScanReport,*,*][ClusterBackgroundScanReport,*,*][ClusterRole,*,kyverno:*][ClusterRoleBinding,*,kyverno:*][ServiceAccount,kyverno,kyverno][ConfigMap,kyverno,kyverno][ConfigMap,kyverno,kyverno-metrics][Deployment,kyverno,kyverno][Job,kyverno,kyverno-hook-pre-delete][NetworkPolicy,kyverno,kyverno][PodDisruptionBudget,kyverno,kyverno][Role,kyverno,kyverno:*][RoleBinding,kyverno,kyverno:*][Secret,kyverno,kyverno-svc.kyverno.svc.*][Service,kyverno,kyverno-svc][Service,kyverno,kyverno-svc-metrics][ServiceMonitor,kyverno,kyverno-svc-service-monitor][Pod,kyverno,kyverno-test]"
   webhooks: '[{"namespaceSelector": {"matchExpressions": [{"key":"kubernetes.io/metadata.name","operator":"NotIn","values":["kyverno"]}]}}]'
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
+  name: kyverno-metrics
+  namespace: kyverno
   labels:
-    app.kubernetes.io/component: kyverno
```

Comment:
This label is reserved to fetch kyverno resources:

https://github.com/kyverno/kyverno/blob/fd0699c6d4411d701c3108147b2f6bf4ddffe9f4/pkg/utils/annotations.go#L18

Can we keep it?

Reply:
Technically it's not part of the `kyverno` component, config is shared across all Kyverno components.
Shall we change the label instead ? What is `KyvernoComponentLabel` used for ?

Reply:
It is used for fetching the Kyverno clusterrole to set webhooks' ownerReference. And we also consider using it to identify Kyverno resources for https://github.com/kyverno/kyverno/issues/5986.

I'm fine if we change to use another label, as long as it is static (can't be configurable via Helm).

Reply:
For the cluster role owners this is fine, as the cluster roles belong to the `kyverno` component, for https://github.com/kyverno/kyverno/issues/5986 we can discuss it later IMHO.